### ISSUE TITLE
Reword the "Unapproved sentences" box

### ISSUE THREAD

**Issue by alanfgh** [Tatoeba/tatoeba2#443]
CK writes:

I wonder about the need for this on every sentence page.

"About unapproved sentences"

At the time the "red" was being introduced, it may have been useful. 

What is likely more confusing to new visitors is the "grey" indirect links.

I'd suggest something more concise and simpler. 

About Color Coding.

1. Red = Unapproved. Please don't translate these. (More Info)

2. Grey = Indirect Translations.  These may not be good matches. (More Info)

Have the "More Info" clickable to a longer explanation (possibly already available in the FAQ -- I didn't check).

Maybe someone else can suggest a clearer or more concise wording.

**Comment by trang** [comment]
I've completely removed the text on the sidebar. The warning icon should give enough information for the user to understand there is something problematic with the sentence.

The main problem that remains is that incorrect sentences are mixed with non CC-BY sentences, which is not ideal, since a non CC-BY sentence can be correct, but because of the warning icon, some users may think there is a mistake in the sentence. But this should be dealt in another issue.
